pulp_container/app/viewsets.py

```diff
@@ -114,7 +116,50 @@ class Meta:
         }
 
 
-class TagViewSet(ReadOnlyContentViewSet):
+Repo = namedtuple("Repo", "type perm")
+mirror_repo = Repo("container.container", "container.view_containerrepository")
+push_repo = Repo("container.container-push", "container.view_containerpushrepository")
+
+
+class ContainerQuerySetViewSet(viewsets.GenericViewSet):
```

Comment:
```suggestion
class ContainerContentQuerySetMixin():
```
Not sure if this works properly with `super()`, but worth a try IMHO.